src/redux/actions/instructorActions.js

```diff
@@ -125,6 +125,21 @@ export const getNewsFeeds = idToken => dispatch => {
   }
 };
 
+export const getInstructors = idToken => async dispatch => {
+  try {
+    const res = await axiosWithAuth(idToken).get(`profiles/role/3`);
+    dispatch({
+      type: GET_INSTRUCTORS,
```

Comment:
THANK YOU for going back and finding the missing code, I know this caused many people a headache after updating their main repo (myself included!)

Reply:
Hey, an extra PR for me and some reviews for you guys lol Sorry, was driving all day, just saw this.